package.json

```diff
@@ -1,25 +1,30 @@
 {
   "name": "screeps-bot-tooangel",
-  "version": "1.1.15",
+  "version": "1.2.0",
```

Comment:
We discussed it a bit in the slack channel. The idea is to announce a release, stop merging PRs. Create a release branch which is tested on private servers and increase the version, when it is fine, release branch is merged, the bot is released. Continue as usual.

I would take this branch as last merge before releasing. So I would suggest to remove the version update and I will start a release after merging and increase the version.